conform-pr/main.go

```diff
@@ -124,18 +141,17 @@ type pullRequest struct {
 	values map[string]string
 }
 
-// checkTitle checks if PR's title does not end with dot.
+// checkTitle checks if PR's title does not end with dot and returns a summary list for checks.
```

Comment:
That's no longer true. Let's revert this change

Reply:
done.